src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.cs

```diff
@@ -5,19 +5,66 @@ namespace System.Diagnostics.Tracing
 {
     /// <summary>
     /// NativeRuntimeEventSource is an EventSource that represents the ETW/EventPipe events emitted by the native runtime.
-    /// Most of NativeRuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider.
+    /// Most of NativeRuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider and will throw a NotImplementedException without hand-written overloads of the Events
+    /// To have a runtime event be fired from the managed code, you need to add a managed definition for that event
+    /// and QCall into the native runtime that invoke the appropriate native sinks for the platform (i.e. ETW, EventPipe, LTTng).
+    /// To see some examples of this, refer to NativeRuntimeEventSource.PortableThreadPool.cs.
```

Comment:
Also we should add that they need to update the getRuntimeEventSources.py to make sure that script doesn't autogenerate a duplicate method that throws NotImplementedException.